posts/2022-01-20-beta-release-post.adoc

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "Support for new changes to SSL certificates in Open Liberty 22.0.0.2-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Support for new changes to SSL certificates in Open Liberty 22.0.0.2-beta - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+blog_description: Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Support for new changes to SSL certificates in Open Liberty 22.0.0.2-beta
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+
+[#allbeta]
+== All Beta Features package
+The All Beta features package for link:{url-about}[Open Liberty] contains all Open Liberty beta features and GA features and functions.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try the beta package, <<feedback, let us know what you think>>.
+
+The following beta features are available in 22.0.0.2-beta:
+
+* <<ssl, Support new change in the SSL certificate with urlencoded string>>
+
+[#ssl]
+=== Support new change in the SSL certificate with urlencoded string
+
+An `X.509` certificate is a digital certificate which is used to manage identity and security in internet communications and computer networking.
+
+The `X.509` certificate can be sent by the intermediate servers (like web server, proxy server or load balancer) to an application server. Due to the deprecation of long lines folding in the `HTTP` header continuation, some of the open source intermediate servers start sending a client certificate in the Privacy-Enhanced Mail (`PEM`) url-encoded certificate to application server.
```

Comment:
Due to the deprecation of long lines folding in the `HTTP` header continuation, some of the open source intermediate servers start sending a client certificate in the Privacy-Enhanced Mail (`PEM`) url-encoded certificate to application server.

not sure of the meaning here but this is awkward- does it mean

Due to the deprecation of long lines folding in the `HTTP` header continuation, some open source intermediate servers now send a client certificate in the Privacy-Enhanced Mail (`PEM`) url-encoded certificate.
?

Reply:
@pnicolucci would you be able to verify that the bottom sentence is still correct?

Reply:
This looks good to me. I ran it by @pmd1nh  as well and he was ok with the change.

Reply:
can we say "some open source intermediate servers **can** now send a client certificate in the Privacy-Enhanced Mail (PEM) url-encoded certificate." ?

since this is  not a default form.

Reply:
is the x509 itself sent in .pem format? Or is the x509 embedded in another certificate that is PEM?

Reply:
the x509 itself is sent in .pem format

Reply:
ok- in that case I'd recommend

Due to the deprecation of long lines folding in the HTTP header continuation, some open source intermediate servers might a client certificate in the Privacy-Enhanced Mail (PEM) url-encoded format.

Reply:
"might a client certificate"  should be "might send a client certificate"

Reply:
sorry, fixed